.drone.star

```diff
@@ -929,6 +978,89 @@ def acceptance():
 
 	return pipelines
 
+def sonarAnalysis(ctx, phpVersion = '7.4'):
+	result = {
+		'kind': 'pipeline',
+		'type': 'docker',
+		'name': 'sonar-analysis',
+		'workspace' : {
+			'base': '/drone',
+			'path': 'src'
+		},
+		'steps':
+			cacheRestore() +
+			composerInstall(phpVersion) +
+			yarnInstall(phpVersion) +
```

Comment:
`make install-nodejs-deps` in `yarnInstall` is not going to work in apps. That make target does not exist in any app. Very few apps run JS tests with coverage. For now, we should get data_exporter working (which has just PHP tests).
You can delete `yarnInstall` - it will be easy to add it in to an app that has JS tests and then get it working.

Reply:
done